### ISSUE TITLE
dropout

### ISSUE THREAD

**Issue by harristeague** [leaderj1001/MobileNetV3-Pytorch#8]
It doesn't look like the dropout parameter is implemented.  Have you trained with dropout (as mentioned in the paper).

**Comment by jlhou** [comment]
I have the same idea with you!May be it did't.

> It doesn't look like the dropout parameter is implemented. Have you trained with dropout (as mentioned in the paper).

**Comment by leaderj1001** [comment]
I'm so sorry. I could not confirm it because I did not have an Issue notification. I really sorry. T.T
I will reflect it as soon as possible. It was a part I forgot.
Thank you !!

**Comment by leaderj1001** [comment]
Again, I am so sorry for the delay. T.T

**Comment by leaderj1001** [comment]
Google's MobileNet-V2 code shows dropouts applied only to classifiers. So I have applied it, and I will try adding a dropout to the MobileBlock.
Thank you !!
